src/pages/shipping.js

```diff
@@ -245,6 +246,8 @@ export const startShipping = async (userName) => {
     // addEventBarCodeScanner('masterSpecimenIdBarCodeBtn', 0, 14, 0);
     addEventModalAddBox(userName);
     hideAnimation();
+    await sleep(400);
+    window.confirm("This shipment is now finalized; no other changes can be made");
```

Comment:
I was wondering if you were getting the window confirm modal at the beginning of the shipping dashboard page.